extension/main.js

```diff
@@ -184,6 +185,29 @@ omnibox.addPrefixQueryEvent("#", {
     }
 });
 
+omnibox.addPrefixQueryEvent("?", {
+    defaultSearch: true,
```

Comment:
I think we'd better keep `defaultSearch` as false. What `defaultSearch` means we search without any prefix sign, just like std docs, and crates, etc. This suit for the high-frequency search cases. :)

Reply:
Ah I see, I wasn't sure what they meant so I just copied them over, will remove.